content/influxdb/v2.4/get-started/process.md

```diff
@@ -3,31 +3,1269 @@ title: Get started processing data
+| 2022-01-01T09:00:00Z | home         | Living Room | temp   | 21.4   |
+| 2022-01-01T10:00:00Z | home         | Living Room | temp   | 21.8   |
+
+{{% /tab-content %}}
+{{% tab-content %}}
+
+When ungrouped, a data is returned in a single table.
+
+{{% note %}}
+`_start` and `_stop` columns have been omitted.
+{{% /note %}}
+
+{{% flux/group-key "[]" true %}}
+
+| _time                | _measurement | room        | _field | _value |
+| :------------------- | :----------- | :---------- | :----- | -----: |
+| 2022-01-01T08:00:00Z | home         | Kitchen     | hum    |   35.9 |
+| 2022-01-01T09:00:00Z | home         | Kitchen     | hum    |   36.2 |
+| 2022-01-01T10:00:00Z | home         | Kitchen     | hum    |   36.1 |
+| 2022-01-01T08:00:00Z | home         | Kitchen     | temp   |     21 |
+| 2022-01-01T09:00:00Z | home         | Kitchen     | temp   |     23 |
+| 2022-01-01T10:00:00Z | home         | Kitchen     | temp   |   22.7 |
+| 2022-01-01T08:00:00Z | home         | Living Room | hum    |   35.9 |
+| 2022-01-01T09:00:00Z | home         | Living Room | hum    |   35.9 |
+| 2022-01-01T10:00:00Z | home         | Living Room | hum    |     36 |
+| 2022-01-01T08:00:00Z | home         | Living Room | temp   |   21.1 |
+| 2022-01-01T09:00:00Z | home         | Living Room | temp   |   21.4 |
+| 2022-01-01T10:00:00Z | home         | Living Room | temp   |   21.8 |
+
+{{% /tab-content %}}
+{{< /tabs-wrapper >}}
+
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+## Aggregate or select specific data
+
+Use Flux [aggregate](/{{< latest "flux" >}}/function-types/#aggregates)
+or [selector](/{{< latest "flux" >}}/function-types/#selectors) functions to
+return aggregate or selected values from **each** input table.
 
-<!--
-### Process data
+```js
+from(bucket: "get-started")
+    |> range(start: 2022-01-01T08:00:00Z, stop: 2022-01-01T20:00:01Z)
+    |> filter(fn: (r) => r._measurement == "home")
+    |> filter(fn: (r) => r._field == "co" or r._field == "hum" or r._field == "temp")
+    |> mean()
+```
 
-Use InfluxDB tasks to process and downsample data. See [Process data](/influxdb/v2.4/process-data/).
+{{% note %}}
+#### Aggregate over time
 
-### Monitor and alert
+If you want to query aggregate values over time, this is a form of
+[downsampling](#downsample-data).
+{{% /note %}}
 
-Monitor your data and sends alerts based on specified logic.
-See [Monitor and alert](/influxdb/v2.4/monitor-alert/).
--->
+### Aggregate functions
+
+[Aggregate functions](/{{< latest "flux" >}}/function-types/#aggregates) drop
+columns that are **not** in the [group key](/flux/v0.x/get-started/data-model/#group-key)
+and return a single row for each input table with the aggregate value of that table.
+
+#### Aggregate examples
+
+{{< expand-wrapper >}}
+
+{{% expand "Calculate the average temperature for each room" %}}
+
+Using the [data written in Get started writing to InfluxDB](/influxdb/v2.4/get-started/write/#view-the-written-data):
```

Comment:
add quotes to denote that this is a separate article/section - "Get started writing data" section